internal/compiler/widgets/material/button.slint

```diff
@@ -111,19 +112,20 @@ export component Button {
         drop-shadow-offset-y: 1px;
     }
 
-    base := MaterialButtonBase {
-        layout-padding-left: 24px;
-        layout-padding-right: 24px;
-        height: 100%;
-        border-radius: background.border-radius;
-        text-color: root.primary ? MaterialPalette.accent-foreground : MaterialPalette.control-foreground;
-        text-opacity: 1.0;
-        
-        clicked => {
-            if root.checkable {
-                root.checked = !root.checked;
+    layout := HorizontalLayout {    
+        base := MaterialButtonBase {
```

Comment:
Wouldn't it be simpler if MaterialButtonBase inherited from HorizontalLayout? Or is there a use of MBB elsewhere where that would be wrong?